src/app/tab/tabs/tabs.component.ts

```diff
@@ -0,0 +1,28 @@
+import { AfterContentInit, Component, ContentChildren, QueryList } from '@angular/core';
+import { TabItemComponent } from '../tab-item/tab-item.component';
+
+@Component({
+  selector: 'tabs',
+  standalone: true,
+  imports: [TabItemComponent],
+  templateUrl: './tabs.component.html',
+})
+export class TabsComponent implements AfterContentInit {
+  @ContentChildren(TabItemComponent) tabs!: QueryList<TabItemComponent>;
+
+  public ngAfterContentInit() {
+    const activeTabs = this.tabs.filter(tab => tab.active());
+
+    if (activeTabs.length === 0) {
+      this.selectTab(this.tabs.first);
+    } else if (activeTabs.length > 1) {
+      this.selectTab(activeTabs[0]);
+    }
+  }
+
+  public selectTab(selectedTab: TabItemComponent) {
+    this.tabs.filter(tab => tab.active()).forEach(tab => tab.active.set(false));
+    selectedTab.active.set(true);
+    selectedTab.tabSelected.emit();
```

Comment:
I thought this reaching into the tab item to emit an event was a little unexpected. I wonder if it might be a good use case for adding an `effect()` on the tab item that responds to the `active()` signal changes and emits when true?